src/components/bookmark-button/helpers/connected-bookmark-button.tsx

```diff
@@ -22,13 +22,13 @@ import makeBookmarkToast from '../toast/make-bookmark-toast'
 
 export function Connected(BookmarkComponent: React.FC<BookmarkButtonProps>) {
 	return function ConnectedBookmarkComponent({
-		tutorialId,
+		tutorial,
 	}: {
-		tutorialId: Tutorial['id']
+		tutorial: { id: Tutorial['id']; name: Tutorial['name'] }
```

Comment:
Refactored this to accept a `tutorial` object since we need both the name and the id for the interactions.